<ui version="4.0" >
 <class>CGexConnectionTestDialog_base</class>
 <widget class="QDialog" name="CGexConnectionTestDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connection test</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="gex_email.qrc" >
    <normaloff>:/gex_email/images/resources/galaxy_icon.png</normaloff>:/gex_email/images/resources/galaxy_icon.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="editSmtpServer" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="editSmtpPort" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="labelSmtpPort" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>SMTP port:</string>
       </property>
       <property name="buddy" >
        <cstring>editSmtpPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="labelSmtpServer" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>SMTP server:</string>
       </property>
       <property name="buddy" >
        <cstring>editSmtpServer</cstring>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="2" >
      <widget class="QLabel" name="labelGalaxyLogo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="gex_email.qrc" >:/gex_email/images/resources/galaxy_logo.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetMessages" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Close dialog</string>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAbort" >
       <property name="toolTip" >
        <string>Abort connection test</string>
       </property>
       <property name="text" >
        <string>Abort</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonAbort</tabstop>
  <tabstop>editSmtpServer</tabstop>
  <tabstop>editSmtpPort</tabstop>
  <tabstop>listWidgetMessages</tabstop>
 </tabstops>
 <resources>
  <include location="gex_email.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonAbort</sender>
   <signal>clicked()</signal>
   <receiver>CGexConnectionTestDialog_base</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>351</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>CGexConnectionTestDialog_base</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>121</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
